packages/web-twig/src/Resources/components/TextField/README.md

```diff
@@ -12,13 +12,28 @@ Advanced example usage:
 
 ```html
 <TextField
+  hasPasswordToggle
   id="example2"
   isRequired
   label="Password"
   messsage="validation failed"
   name="example2"
   placeholder="Placeholder"
-  type="password"
+  validationState="error"
+/>
+```
+
+or
```

Comment:
I think it will be better to know the difference between them. Eg. I would prefer adding a subtitle with `Default TextField`, `TextField with password toggle (button to reveal password)`. What do you think?

Reply:
ok, why not :)